core-bundle/src/Resources/contao/library/Contao/Environment.php

```diff
@@ -324,7 +324,7 @@ protected static function script()
 	 */
 	protected static function request()
 	{
-		return preg_replace('/^' . preg_quote(static::get('path'), '/') . '\/?/', '', static::get('requestUri'));
+		return static::get('requestUri');
```

Comment:
```suggestion
		trigger_deprecation('contao/core-bundle', '5.0', 'Using "%s()" has been deprecated and will no longer work in Contao 6.0. Use "%s::%s" instead.', __METHOD__, __CLASS__, 'requestUri');

		return preg_replace('/^' . preg_quote(static::get('path'), '/') . '\/?/', '', static::get('requestUri'));
```
I think we should not change these methods, but instead deprecate them and replace all usages of `Environment::get('request')` with `Environment::get('requestUri')`

Reply:
Usages have been replaced in fc5b8d1081f62fbfb51f3e4ded9516ccac49dbd3.